client/components/gridicon/docs/example.jsx

```diff
@@ -19,7 +19,8 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<div>
+			<div className="design-assets__group">
```

Comment:
I'm adding another icon currently — do I just need to remove the classnames so it's a classless div?